### DISCUSSION TITLE
Win系统建议使用冰凌输入法平台，兼容性超好！

### DISCUSSION THREAD

**Discussion by ghost** [amorphobia/openfly#57] [一般讨论/General]
冰凌输入法(平台)是极点、加加、多多、小小、搜狗、Rime等平台的最新最好替代，对各种码表和系统的兼容性非常好！
冰凌输入法(平台)釆用TSF输入法框架和COM编程规范开发，尤其与Windows10、11操作系统有较高的契合度和适应性，以UNICODE为编码内核，支持CJK字符集的所有汉字输入，支持桌面应用程序和应用商店应用程序。
内置五笔和拼音，可以在“系统词库管理”导入各种词库码表TXT，如自然码/小鹤音形、仓颉五代等等。只需写好码表头，很简单。
兼容多种输入习惯，支持特殊符号、日期时间、数字转换等便捷输入和按键、快捷键定义等功能。
支持单字输入和单字优先功能，提供自动调频、手动调频、手工置顶等调频方法。
词库开放，个性化词库定制，也为第三方词库挂接提供了接口。
纯码输入、混合输入、临时拼音、临时英文等多种文字输入方式。
便捷输入、联想输入、重复输入、顶字上屏等输入方法。
无插件、无广告、无弹窗、无捆绑、不骚扰，免费。
![ui](https://github.com/amorphobia/openfly/assets/47594662/efe1c68c-ae70-4464-ac26-530e2842e848)

纯净小巧，安装包才2.4MB ！！！
[www.icesofts.com](https://www.icesofts.com/)

**Comment by ghost** [comment]
官群有一份好的小鹤大词库码表，以tab分隔，改自[wubi-lex](https://githubfast.com/aardio/wubi-lex)。

[CODETABLEHEADER]
Name=小鹤音形[大词库]
Version=10
Author=何某
CodeScheme=小鹤音形
CodeLength=4
BWCodeLength=0
SpecialPrefix=0
PhraseRule=3
pa2=w11+w12+w21+w22
pa3=w11+w21+w31+w32
pe4=w11+w21+w31+w41
[CODETABLE]
a	啊
……

**Comment by ze-mou** [comment]
省略号输入的就是码表里面的信息是吗 官鹤没有找到大码表 能否为我提供一份呢
